### PR TITLE
DS-504 Update display value of ButtonGroup to inline-flex

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: src/components/buttons/ButtonGroup.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ export function ButtonGroup({
 }
 
 const StyledButtonGroup = styled.span`
+  display: inline-flex;
```
Comment (david-cho-lerat-sonarsource) [inline src/components/buttons/ButtonGroup.tsx:44]: NP: why change the order?

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
:+1:

File: src/components/buttons/ButtonGroup.tsx (hunk 1 above)
Comment (gregaubert) [reply src/components/buttons/ButtonGroup.tsx:44]: I usually like to have the display value first and follow with the CSS properties that change it.

Comment (gregaubert) [reply src/components/buttons/ButtonGroup.tsx:44]: like here, the align-items and gap are properties that are tied to display: inline-flex, and change the way the inline flex looks.

Comment (david-cho-lerat-sonarsource) [reply src/components/buttons/ButtonGroup.tsx:44]: ok

**Pull request merged**